File: src/main/scala/util/Configuration.scala
package util

import com.mongodb.casbah.Imports._

object Configuration {

  lazy val additionalFiles = sys.props.get("additionalFiles").getOrElse("/Users/xbucchiotty/Documents/Travail/Scala/XPUA/xpua/AdditionalFiles")
  lazy val mongohost = sys.props.get("mongo.host").getOrElse("localhost")
  lazy val mongoClient = MongoClient(Configuration.mongohost)
  lazy val db = mongoClient("sixtheam")
}

File: src/main/java/util/Configuration.java
package util;

import com.mongodb.DB;
import com.mongodb.Mongo;

import java.net.UnknownHostException;

public abstract class Configuration {

    public static final String additionalFiles = System.getProperty("additionalFiles", "AdditionalFiles/");

    public static final String mongohost = System.getProperty("mongo.host", "localhost");

    public static DB getDb() throws UnknownHostException {
        return new Mongo(mongohost).getDB("xpua");
    }
}
